runtime/filetype.vim

```diff
@@ -233,9 +233,9 @@ au BufNewFile,BufRead *.fb			setf freebasic
 
 " Batch file for MSDOS. See dist#ft#FTsys for *.sys
 au BufNewFile,BufRead *.bat			setf dosbatch
-" *.cmd is close to a Batch file, but on OS/2 Rexx files also use *.cmd.
+" *.cmd is close to a Batch file, but on OS/2 Rexx files and TI DSP linker command files also use *.cmd.
 au BufNewFile,BufRead *.cmd
-	\ if getline(1) =~ '^/\*' | setf rexx | else | setf dosbatch | endif
+	\ if getline(1) =~ '/\*\s\+\S\+\.cmd' | setf cmd | elseif getline(1) =~ '^/\*' | setf rexx | else | setf dosbatch | endif
```

Comment:
Your example file has the script name on line 2. Also I find this is a strange condition to detect TI DSP file. I can imagine that batch-scripts may also have similar comments which may match `*.cmd`. Also I find cmd as filetype a bit too generic. How does this related to "TI Linker"?

Reply:
dosbatch will not use `/* XXX.cmd` because `/* XXX */` is comment of tild.